pyshelf/cloud/factory.py

```diff
@@ -13,10 +12,6 @@ def create_storage(self, bucket_name):
         # required and is not used because we want the ability to change
         # buckets when we want.
         bc = utils.get_bucket_config(self.config, bucket_name)
-        if bc is None:
```

Comment:
I am leaving this out since bulk update will not use this. It currently ignores non-existent bucket and when this issue is fixed it won't get to this point before throwing an error.